pkg/kubelet/cm/dra/manager.go

```diff
@@ -28,50 +28,40 @@ import (
 	"k8s.io/dynamic-resource-allocation/resourceclaim"
 	"k8s.io/klog/v2"
 	dra "k8s.io/kubernetes/pkg/kubelet/cm/dra/plugin"
+	"k8s.io/kubernetes/pkg/kubelet/cm/dra/state"
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 )
 
+// draManagerStateFileName is the file name where dra manager stores its state
+const draManagerStateFileName = "dra_manager_state"
+
 // ManagerImpl is the structure in charge of managing DRA resource Plugins.
 type ManagerImpl struct {
-	// cache contains cached claim info
-	cache *claimInfoCache
+	// checkpoint cache contains cached claim info
+	cache state.CheckpointState
```

Comment:
if we are going to keep a 1:1 mapping of cache--> state, then I'd rather rename this variable to `state`. Otherwise we should have a new type that includes the state, alongside any other generated values (e.g. annotations) and call that the cache.